services/storage/src/simcore_service_storage/db_file_meta_data.py

```diff
@@ -77,8 +77,14 @@ async def list_filter_with_partial_file_id(
 ) -> list[FileMetaDataAtDB]:
     stmt = sa.select(file_meta_data).where(
         (
-            (file_meta_data.c.user_id == f"{user_id}")
-            | file_meta_data.c.project_id.in_(f"{pid}" for pid in project_ids)
+            sa.or_(
+                (
+                    file_meta_data.c.user_id.is_(user_id)
```

Comment:
did not know that is_ also worked with integers like that. cool.

Reply:
what is the advantage of ```sa.or_``` over ```|```  if any?

Reply:
> did not know that is_ also worked with integers like that. cool.

Yeah, seems it didn't work 😓. In the end I had to use `==`.

Reply:
> what is the advantage of `sa.or_` over `|` if any?

For some reason I couldn't get the python-style bool statement to work when using `|`. So it would complain about the 
```python
if user_id is not None else True
```